src/library/export/dlglibraryexport.cpp

```diff
@@ -231,13 +231,13 @@ void DlgLibraryExport::checkExistingDatabase() {
             m_pVersionCombo->clear();
             m_pVersionCombo->setEnabled(true);
             for (int versionIndex = 0;
-                    versionIndex < static_cast<int>(e::all_versions.size());
+                    versionIndex < static_cast<int>(e::supported_schemas.size());
                     ++versionIndex) {
-                e::engine_version version = e::all_versions[versionIndex];
+                e::engine_schema schemaVersion = e::supported_schemas[versionIndex];
                 m_pVersionCombo->insertItem(0,
-                        QString::fromStdString(version.name),
+                        QString::fromStdString(to_application_version_string(schemaVersion)),
```

Comment:
A breaking change introduced in recent libdjinterop is that the "versions" that it works with are now aligned with the database schema (i.e. the Engine Library), rather than the marketing versions of the apps that use the schema (Engine OS, Engine DJ).

This is a more reliable model.  For human-friendliness, the new `to_application_version_string()` provides the human-readable description of the apps associated with this schema.

Reply:
The schema isn't exposed to the end user - they will see a friendly name in the drop-down as before:
![image](https://github.com/user-attachments/assets/16d82326-ffbb-44c6-9cc5-f3d91a96b489)